opentelemetry-aws/Cargo.toml

```
@@ -25,17 +25,17 @@ detector-aws-lambda = ["dep:opentelemetry-semantic-conventions"]
 internal-logs = ["tracing"]
 
 [dependencies]
-opentelemetry = { version = "0.28" }
-opentelemetry_sdk = { version = "0.28", optional = true }
-opentelemetry-semantic-conventions = { version = "0.28", optional = true, features = [
+opentelemetry = { version = "0.29" }
```

please set everywhere `workspace = true` instead of `version = "0.29"`
```suggestion
opentelemetry = { workspace = true }
```

---

done!